console/models/main.py

```diff
@@ -220,6 +220,7 @@ class Meta:
     event_id = models.CharField(max_length=32, null=True, blank=True, help_text="介质同步事件ID,弃用，使用表service_share_record_event")
     share_version = models.CharField(max_length=15, null=True, blank=True, help_text="应用组发布版本")
     share_version_alias = models.CharField(max_length=32, null=True, blank=True, help_text="应用组发布版本别名")
+    share_app_version_info = models.CharField(max_length=255, help_text="应用组发布版本描述")
```

Comment:
version detail save to version model.

Reply:
This field is mainly used for cloud application stores to reduce the number of cyclic API calls